app/views/bulk_upload_lettings_resume/confirm.html.erb

```diff
@@ -5,14 +5,23 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <span class="govuk-caption-l">Bulk upload for lettings (<%= @bulk_upload.year_combo %>)</span>
-    <h1 class="govuk-heading-l">You have chosen to upload all logs from this bulk upload.</h1>
+    <h1 class="govuk-heading-l">Are you sure you want to upload all logs from this bulk upload?</h1>
 
-    <p class="govuk-body"><%= logs_and_errors_warning(@bulk_upload) %></p>
+    <p class="govuk-body">
+      <%= logs_and_errors_warning(@bulk_upload) %>
+      <%= govuk_link_to "View the error report", deletion_report_bulk_upload_lettings_result_path %>
```

Comment:
I would have expected this to be the full error report instead of the deletion one, are we meant to link to it twice?

Reply:
me too, the designs have been clarified, you're right :D